packages/access-api/src/models/accounts.js

```diff
@@ -0,0 +1,113 @@
+// @ts-ignore
+// eslint-disable-next-line no-unused-vars
+import * as Ucanto from '@ucanto/interface'
```

Comment:
should we import types needed like below DelegationRecord instead of importing directly in source and ignoring eslint error?

Reply:
perhaps we could disable `no-unused-vars` since it is not ts-aware and instead only use `@typescript-eslint/no-unused-vars`. https://stackoverflow.com/a/61555310

Reply:
@vasco-santos the interface package is made with an empty .js file to enable imports like this, because generic types in jsdoc typedefs are a pain.

@gobengo that could work and it looks cleaner

Reply:
fwiw what I proposed before doesn't work, but this does https://stackoverflow.com/a/66261985

Reply:
https://github.com/web3-storage/w3protocol/pull/409/files#r1092603863